pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -335,6 +335,8 @@ func manageServiceAccountCABundle(ctx context.Context, lister corev1listers.Conf
 	requiredConfigMap, err := resourcesynccontroller.CombineCABundleConfigMaps(
 		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.TargetNamespace, Name: "serviceaccount-ca"},
 		lister,
+		"kube-scheduler",
+		"",
```

Comment:
leave for another PR, but I really look forward to learning what this is for.  I don't remember it.